packages/eslint-plugin-office-addins/src/rules/test-for-null-using-isNullObject.ts

```diff
@@ -0,0 +1,155 @@
+import { TSESTree, AST_NODE_TYPES } from "@typescript-eslint/typescript-estree";
```

Comment:
You said you would look at the [no-use-before-define](https://github.com/eslint/eslint/blob/85a2725b1fade5538e727102d9701ccb503e54d4/lib/rules/no-use-before-define.js) rule before submitting. Did you have a look in this yet?

Reply:
I wasn't intending to rewrite this with that different pattern style.  Do you think I should?

Reply:
Nahh, I think it is fine :D